kimhaejung/models/likeDao.js

```diff
@@ -0,0 +1,23 @@
+const { appDataSource } = require("./dataSource");
+
+const likePosts = async (userId, postId) => {
+  try {
+    console.log(333);
+    return await appDataSource.query(
+      `
+      INSERT INTO likes(
+        user_id,
+        post_id
+        ) VALUES ( ?, ? );
+      `,
+      [userId, postId]
+    );
+  } catch (err) {
+    console.log(err);
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 500;
```

Comment:
상태코드 500은 backend가 에러처리를 안 했다는 의미이기 때문에 500 대신에 400번대 상태코드를 사용해 주세요.
(이 부분도 전체적으로 확인해 주세요.)
```suggestion
    error.statusCode = 400;
```